lib/bundler/templates/newgem/spec/spec_helper.rb.tt

```diff
@@ -1,2 +1,11 @@
-$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
+require "bundler/setup"
 require "<%= config[:namespaced_path] %>"
+
+RSpec.configure do |config|
+  # Enable flags like --only-failures and --next-failure
+  config.example_status_persistence_file_path = ".rspec_status"
+
+  config.expect_with :rspec do |c|
+    c.syntax = :expect
+  end
```

Comment:
oops, copied from the bundler specs. suppose I should've put it in another commit, but I do think it's a good idea to include it